content/en/docs/concepts/security/_index.md

```diff
@@ -3,4 +3,241 @@ title: "Security"
 weight: 85
 description: >
   Concepts for keeping your cloud-native workload secure.
+simple_list: true
 ---
+
+This section of the Kubernetes documentation aims to help you learn to run
+workloads more securely, and about the essential aspects of keeping a
+Kubernetes cluster secure.
+
+Kubernetes is based on a cloud-native architecture, and draws on advice from the
+{{< glossary_tooltip text="CNCF" term_id="cncf" >}} about good practice for
```

Comment:
```suggestion
{{< glossary_tooltip text="CNCF" term_id="cncf" >}} about good practices for
```

Reply:
I could make this change, but: I'm also happy with the original wording. It doesn't feel like this is a blocker to merging the PR.